### PR TITLE
feat:Site Navigation icon on Manage spaces - EXO-65324 - Meeds-io/MIPs#68

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

**Review by azayati — APPROVED** [review]
LGTM

File: component/service/src/main/java/org/exoplatform/social/rest/api/EntityBuilder.java [hunk 1]
```diff
@@ -630,6 +634,15 @@ public static SpaceEntity buildEntityFromSpace(Space space, String userId, Strin
             spaceEntity.setUnreadItems(unreadItems);
           }
         }
+        if (expandFields.contains(RestProperties.NAVIGATIONS_PERMISSION)) {
+
+
+
```
Comment (azayati) [inline component/service/src/main/java/org/exoplatform/social/rest/api/EntityBuilder.java]: remove empty lines

**Review by azayati — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 export function getSpaces() {
-  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount`, {credentials: 'include'}).then(resp => resp.json());
+  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount,navigations_permission`, {credentials: 'include'}).then(resp => resp.json());
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js:2]:
```suggestion
  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount, navigations_permission`, {credentials: 'include'}).then(resp => resp.json());
```

File: webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 }
 
 export function searchSpaces(search) {
-  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?q=${search}&sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount`, {credentials: 'include'}).then(resp => resp.json());
+  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?q=${search}&sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount,navigations_permission`, {credentials: 'include'}).then(resp => resp.json());
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js:6]:
```suggestion
  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?q=${search}&sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount, navigations_permission`, {credentials: 'include'}).then(resp => resp.json());
```

File: webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 }
 
 export function getSpacesPerPage(offset) {
-  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?offset=${offset}&sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount`, {credentials: 'include'}).then(resp => resp.json());
+  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?offset=${offset}&sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount,navigations_permission`, {credentials: 'include'}).then(resp => resp.json());
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js:10]:
```suggestion
  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?offset=${offset}&sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount, navigations_permission`, {credentials: 'include'}).then(resp => resp.json());
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js (hunk 2 above)
Comment (GouadriaHanen) [reply webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js:2]: it needs to be String in expand no need for space here

File: webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js (hunk 3 above)
Comment (GouadriaHanen) [reply webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js:6]: same here

File: webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js (hunk 4 above)
Comment (GouadriaHanen) [reply webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js:10]: same here

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**